scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
 static void *copy_(void *src) { return src; }
 static void free_(void *doc) { }
 
+GEANY_API_SYMBOL
 GType scnotification_get_type(void) {
 	static gsize type_id = 0;
```

Comment:
It might be because [`g_once_init_leave()`](https://developer.gnome.org/glib/unstable/glib-Threads.html#g-once-init-leave) expects a `gsize` value, and if that API expects a pointer to a `gsize`-sized value, it is a lot safer to use a real `gsize` as the pointer and have the cast happen by value, instead of dereferencing a pointer to a `GType`-sized value as if it was a `gsize`-sized one.

It's kinda odd `g_once_init_*()` API claims a `void*` pointer though, so I don't know.